CHANGES_NEXT_RELEASE

```diff
@@ -3,3 +3,4 @@
 - Fix: case-sensitive header duplication (e.g. "Content-Type" and "Content-type") in custom notifications (#2893)
 - Fix: bug in GTE and LTE operations in query filters (q/mq), both for GET operations and subscriptions (#2995)
 - Fix: Wrong "max one service-path allowed for subscriptions" in NGSIv2 subscription operation (#2948)
+- Fix: Add support for accessing response headers when making CORS requests (#3030)
```

Comment:
Could this be considered part of "CORS Preflight Requests support for all NGSIv2 resources"? Or it is better to consider it a separate item of work?

Reply:
Well, it is not directly related with Preflight requests but since we are touching those parts in this PR as well, I can change the line below

> - Add: CORS Preflight Requests support for all NGSIv2 resources, -corsMaxAge switch (#501)

to 

- Add: CORS Preflight Requests support for all NGSIv2 resources, -corsMaxAge switch, CORS exposed headers (#501)

Reply:
Ok, with a small ammedment to include the number of the issue about exposed headers:

> Add: CORS Preflight Requests support for all NGSIv2 resources, -corsMaxAge switch, CORS exposed headers (#501, #3030)

Reply:
Fixed in 3e6e50a